### DISCUSSION TITLE
runelite not accepting gmail loggin.

### DISCUSSION THREAD

**Discussion by Dmoney8730** [runelite/runelite#16240] [Q&A] [answered]
been playing osrs for a few weeks. created my account using gmail. now trying to swtich to runelite and it will not accept my log in information. anyone ran into this or have a fix? wont even let me reset my password. just says there is a problem.

**Comment by Nightfirecat** [comment, marked as answer]
What exactly is saying there is a problem, RuneLite? or Google?

**Reply by Dmoney8730** [reply]
Ok, so on runlite, it says invalid credentials when i enter my information. 
i can however enter my infor into the greendoor with the arrow at the top left. it accepts it but still doesnt log in.
i then clicked recover password at the bottom of log in screen. at that point it redirected me to web browser which when i continued the recovery proccess it said there was a problem recovering the account.

however i can go back to the regular runescape and log in just fine.

**Reply by Dmoney8730** [reply]
Just figured it out. even though i created my account using my gmail my email wasnt linked to my account. went back to osrs app and in setting tabs linked email, it is now allowing me to sign into runelite. thank you for your time.
